_posts/en/newsletters/2022-12-07-newsletter.md

```diff
@@ -0,0 +1,238 @@
+  available by Bitcoin Core [CPFP carve outs][topic cpfp carve out] and
+  used in the LN protocol to ensure that both parties involved in a
+  contract can [CPFP][topic cpfp] fee bump a transaction related to that
+  contract.  Anchor outputs have several downsides---some of them
+  fundamental (see [Newsletter #224][news224 anchors])---but others
+  which can be addressed.
+
+    Ephemeral anchors build on the [v3 transaction relay proposal][topic
+    v3 transaction relay] to allow v3 transactions to include a
+    zero-value output paying a script that is essentially `OP_TRUE`,
+    which permits that transaction to be CPFP fee bumped anyone on the
+    network with a spendable UTXO.  The fee-bumping child transaction
+    can itself be RBF fee bumped by anyone else with a spendable UTXO.
+    In combination with other parts of the v3 transaction relay
+    proposal, it can be hoped that this will eliminate all policy-based
+    concerns about [transaction pinning attacks][topic transaction
+    pinning] against time-sensitive contract protocol transactions.
+
+    Additionally, because anyone can fee bump a transaction containing
+    an ephemeral output, it can be used for contract protocols involving
+    more than two participants.  The existing Bitcoin Core carve out
+    rule only reliably works for two participants and [previous
+    attempts][bitcoin core #18725] to increase it required an
+    arbitrary upper limit on participants.
+
+    Sander's [implementation][bitcoin core #26403] of ephemeral anchors
+    makes it possible to begin testing the idea (along with the other v3
+    transaction relay behaviors previously implemented by that
+    proposal's author).
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Bump unconfirmed ancestor transactions to target feerate][review club 26152-2]
+is a PR by Xekyo (Murch) that improves the accuracy of the wallet's fee
+calculation in the case that unconfirmed UTXOs are selected as inputs.
+Without the PR, the fee is set too low if the
+feerates of some transactions being used as inputs are lower
+than that of the transaction being constructed. The PR fixes
+this by adding enough fee to "pay for" low-fee unconfirmed source
+transactions.
+
+Adjusting the fee to account for these ancestor transactions turns out
+to be similar to choosing which transactions to include
+in a block, so this PR adds a class called `Miniminer`.
+
+This PR review spanned two weeks,
+[Nov23][review club 26152] and [Nov30][review club 26152-2].
+
+{% include functions/details-list.md
+  q0="What problem does this PR address?"
+  a0="The wallet fee estimation doesn't take into account that it must also
+      pay for all unconfirmed ancestors with a lower feerate than the target."
```

Comment:
More precisely, the PR can also detect when it *doesn't* need to pay for ancestors with a lower feerate than the target!
```suggestion
  a0="The wallet fee estimation doesn't take into account that it may also
      need to pay for unconfirmed ancestors with a lower feerate than the target."
```